server/routes/playlist.js

```diff
@@ -253,4 +253,25 @@ router.get('/get-random-playlist', jwt.verifyJWT, async (req, res, next) => {
   }
 })
 
+/* updates all similarity scores between playlists in db */
+router.post('/update-similarity-scores', async (req, res, next) => {
+  try {
+    const allPlaylists = await PlaylistSimilarity.find()
+
+    const promises = allPlaylists.map(async (item) => {
+      const { firstPlaylistId, firstPlaylistVector, secondPlaylistId, secondPlaylistVector } = item
+
+      /* recalculate similarity score */
+      const ownSimilarityScore = await similarity.calculateOwnSimilarity(firstPlaylistVector, secondPlaylistVector)
+
+      /* update entry with new similarity score */
+      await PlaylistSimilarity.findOneAndUpdate({ firstPlaylistId, secondPlaylistId }, { ownSimilarityScore })
+    })
+    await Promise.all(promises)
```

Comment:
Can you move this into a class or util so that it can also be called from a script?